doc/user/content/sql/functions/now_and_mz_now.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "now and mz_now functions"
+description: "In Materialize mz_now() acts the way now() does in most systems."
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+In Materialize, `now()` doesn't represent the system time, as it does in most systems; it represents the time with timezone when the query was executed. It cannot be used when creating views.
+
+`mz_now()` comes closer to what `now()` typically indicates. It represents the logical time at which a query executes. Its typical uses are:
+
+* Internal debugging
+* Defining [temporal filters](/sql/patterns/temporal-filters/)
+
+## Internal debugging
+
+`mz_now()` can be useful if you need to understand the time up to which data has been ingested by `materialized`. It corresponds to the timestamp column of [`SUBSCRIBE`](/sql/subscribe). Generally a `CREATE SOURCE` command will cause Materialize to ingest data and produce timestamps that correspond to milliseconds since the Unix epoch.
+
+## Temporal filters
+
+You can use `mz_now()` to define temporal filters for materialized view, which implement various windowing idioms.
+
+For more information, see [Temporal Filters](/sql/patterns/temporal-filters/).
+
+## Restrictions
+
+You can only use `mz_now()` to establish a temporal filter in one of the following types of clauses:
+
+* WHERE clauses, where `mz_now()` must be directly compared to [`mztimestamp`](/sql/types/mztimestamp) expressions not containing `mz_now()`
+* A conjunction (AND), where `mz_now()` must be directly compared to [`mztimestamp`](/sql/types/mztimestamp) expressions not containing `mz_now()`.
+
+  At the moment, you can't use the `!=` operator with `mz_now` (we're working on it).
```

Comment:
This is mostly duplicated with the temporal filters page, except it is missing some detail. Let's just delete it?

```suggestion
```